### ISSUE TITLE
anaconda environment?

### ISSUE THREAD

**Issue by Casyfill** [instacart/lore#115]
Hey, that looks like a great project and very close to what we were looking for. 
I understand why you guys ditched any environments, but still - is there any way to use lore with existing / specific conda environment (or do you think it is relatively simple to PR this feature?)

**Comment by montanalow** [comment]
There isn't currently a way to specify a pre-existing virtual env, although it wouldn't take much more than adding a new environment variable and using that instead of searching for it. Is that what you had in mind?

**Comment by Casyfill** [comment]
yes! but also - can lore spin out another conda environment, instead of local python?

**Comment by montanalow** [comment]
Lore doesn't use local python per se. It uses `pyenv` to build whatever version of python is specified in runtime.txt or `init --python-version`. Are you advocating for using `conda` as the python installation service rather than `pyenv`? It might help me understand the motivation if you added more context.

**Comment by Casyfill** [comment]
yes, exactly;

**Comment by montanalow** [comment]
Part of the motivation for Lore is to abstract away virtualenv maintenance, so developers don't need to worry about conda vs pyenv etc. There isn't currently any support for conda/environment.yaml or Pipfile, but it wouldn't be impossible to add. For example, do you think Lore should check for `Pipfile`, `environment.yaml` or `requirements.txt` and use whichever it finds first to setup the virtualenv?

**Comment by Casyfill** [comment]
Yes, that makes sense. It would be also nice to specify/pin miniconda
version (or just have it as an option)

**Comment by mturok** [comment]
Out of curiosity, if you aren't using conda explicitly, are you able to pick up the mkl dependencies for numpy in some other fashion?  (eg, specific wheels?)

**Comment by metatron1973** [comment]
Still system wide fraud inside instacarts Wilson score and Tips being paid ... ![image](https://user-images.githubusercontent.com/80199202/179419648-6ab304bd-9465-4c87-9644-4c83ea65f357.png)
